### PR TITLE
Update .travis.yml

### PR DESCRIPTION
**Pull request by dmitry-mukhin**
- add python 3.6
- always test against latest django patch
- celery 4 requires django 1.8+ and python 2.7+, so just stay with 3 for a while

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -6,21 +6,22 @@ python:
...
+  - DJANGO=1.7
+  - DJANGO=1.8
+  - DJANGO=1.9
 
 services:
   - redis-server
 
 install:
-  - pip install -q $DJANGO
+  - pip install -q django~=$DJANGO.0
```
Comment (kencochrane) [inline .travis.yml]: Neat trick, didn't know about that. Thanks

**Comment by dmitry-mukhin** [comment]
err on travis' side :(
```
Collecting coverage>=3.6 (from coveralls)
  Downloading coverage-4.3.4-cp27-cp27mu-manylinux1_x86_64.whl (189kB)
    21% |███████                         | 40kB 4.7MB/s eta 0:00:01Exception:
```

**Comment by kencochrane** [comment]
Error from Travis doesn't look related. Going to merge

File: .travis.yml (hunk 1 above)
Comment (dmitry-mukhin) [inline .travis.yml]: https://www.python.org/dev/peps/pep-0440/#id52

**Pull request merged**